cilium/ebpf.io-summit
# fix: move Charter page to repository ebpf.io

**Describe what changes this pull request brings**

This pull request removes the Charter page. The page will be moved to the repository [ebpf.io](https://github.com/ebpf-io/ebpf.io), you can find the PR of the Charter page [here](https://github.com/ebpf-io/ebpf.io/pull/171).

**Additional changes**
- change the link of Charter in the header menu to `https://ebpf.io/charter` 
- uninstall `gatsby-transformer-remark` since there is no markdown pages anymore


**Steps to test**
1. Open [the page](https://deploy-preview-27--ebpf-summit-2021.netlify.app/charter/)
2. Confirm that everything looks and works as expected
